app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<h4>Log in</h4>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do
+|f| %>
+<div class="mb-3">
+  <%= f.label :email, class:'form-label' %><br />
+  <%= f.email_field :email, autofocus: true, autocomplete: "email",
+  class:'form-control' %>
+</div>
+
+<div class="mb-3">
+  <%= f.label :password, class:'form-label' %><br />
+  <%= f.password_field :password, autocomplete: "current-password",
+  class:'form-control' %>
+</div>
```

Comment:
### OPTIONAL
- [ ] When ı try login unexist user ı am not getting any messages. It will be better if show messages to users.